tests/system-defaults-write.nix

```diff
@@ -1,6 +1,25 @@
 { config, pkgs, lib, ... }:
 
 {
+  imports = [
```

Comment:
Does this test fail without the `type` change above?

Reply:
Yes, and only the `WebKit2DeveloperExtrasEnabled` key shows up in the final config. I pushed only the test changes [to a separate branch](https://github.com/zhaofengli/nix-darwin/actions/runs/13862641673/job/38794597899).